site/documentation/content/api/command-router/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Command Router API Specification"
+linkTitle: "Command Router API"
+weight: 425
+resources:
+  - src: preconditions.svg
+  - src: register_cmd_consumer.svg
+  - src: unregister_cmd_consumer.svg
+  - src: set_last_known_gateway_success.svg
+---
+
+The *Command Router API* is used by *Protocol Adapters* to configure the Command Router service component, enabling it to route command & control messages to the protocol adapters that the target devices are connected to.
```

Comment:
I do not think that the adapters *configure* the component itself, that would be done by environment variables during startup, wouldn't it?
How about
> Protocol Adapters use the *Command Router API* to supply information during run time that a Command Router service component uses to route command & control messages ...

Reply:
Yes, maybe *configure* is a bit confusing here.
I've adapted the proposal slightly to:
> *Protocol Adapters* use the *Command Router API* to supply information with which a Command Router service component can route command & control messages to the protocol adapters that the target devices are connected to.